DFC.App.Account.Services.DSS/Services/DssService.cs

```diff
@@ -61,11 +60,77 @@ public async Task<Customer> CreateCustomerData(Customer customerData)
             return null;
 
         }
-        private StringContent AddDefaultRequestHeaders(StringContent content)
+
+        public async Task<Customer> GetCustomerData(string customerId)
+        {
+            
+            var request = await BuidRequestMessage();
+            request.Headers.Add("version", _dssSettings.Value.CustomerApiVersion);
+
+            var customer = new Customer();
+            
+            customer = await GetCustomerDetail(customerId, request);
+            customer.Addresses = await GetCustomerAddressDetails(customerId, request);
+            customer.Contact = await GetCustomerContactDetails(customerId, request);
+
+            return customer;
+
+        }
+
+        private async Task<HttpRequestMessage> BuidRequestMessage()
```

Comment:
Does this need to be async if it's not doing any async operations?

Reply:
We await the call so we can run it async

Reply:
It doesn't need to be async if it's not doing anything async operations, in fact Resharper is complaining about that fact

Reply:
Happy to approve if there's a future reason it needs to be?